README.md

```diff
@@ -121,6 +123,22 @@ This will generate a coverage.out file in the root directory of the repo which y
 $ go tool cover -html=coverage.out
 ```
 
+#### Code Organization
+All CLI go code lives under the "cmd" directory, who sub-directories map one is to one to the toplevel commands, namely
```

Comment:
in't there an error in this sentence ? `sub-directories map one is to one to the toplevel `

Reply:
I mean in the cmd directory there is a directory for each top level command, that is why I wrote "one to one mapping". Will see how to make that more clear.

Reply:
```suggestion
All CLI go code lives under the "cmd" directory, who sub-directories map one to one to the toplevel commands, namely
```

Reply:
@mkbhanda I am sorry about this accidental commit I have done!  I didn't realize how this (suggested change) github tool works .. tomorrow I will fix it. But you could get what my commit was all about -  I meant to remove this "is"